CKB-Explorer-PRDs/WalletConnect/GeneralComponent.md

```diff
@@ -0,0 +1,43 @@
+# CKB General Component for WalletConnect
+
+## General Component for Wallet Side
+
+A general component is provided for WalletConnect on the side of third party wallet.
+
+### Connecting flowchart
+
+> Scanning QR code is the most used and also recommended for users to connect to dApp with your wallet. We also offer a way to connect by pasting the WalletConnect code which could be used for testing. [UI](https://www.figma.com/file/6XNoimRDbFTTNm016rbIdU/Magickbase?type=design&node-id=16072-38648&t=rF3mLzNYeaveGD6Q-0)
+
+#### UI Sketch
+
+![UI图](pic/2023-05-30-09-20-22.png)
+
+#### Status1: Unconnected
+
+While the dapp is unconnected with the wallet, users can click on the button below to scan QR code on the dapp side in order to connect the dapp.
+
+![UnconnectStatus](pic/2023-05-30-08-19-27.png)
+
+#### Status2: Connecting
+
+After clicking the scan button, and if the QR code is correct then the dapp and wallet are trying to communicate following the protocol. If the QR code is not in correct format, corresponding messages will be popped up.
+
+![ConnectingStatus](pic/2023-05-30-08-22-37.png)
```

Comment:
IMO, the status of disconnection and connecting is quite similar to account model of Ethereum because the interaction is the same:

```
1. connect wallet
2. waiting
3. show wallet/show error message
```

So the current UX of wallet connect can be used